nwb.config.js

```diff
@@ -0,0 +1,23 @@
+const browsers = require('./test/browsers');
```

Comment:
Right, don't get me wrong, for sure easier setup === easier life for maintainers, but if we just used webpack 2, every new contributor could be on same page as it's generely known bundler.  But yeah in the end I don't mind.